bennett/block-template-parts/header-minimal.html

```diff
@@ -0,0 +1,31 @@
+<!-- wp:group {"className":"gapless-group","layout":{"inherit":"true"}} -->
```

Comment:
The only changes I see introduced by this file over the version in /blockbase are spaces.  Can this be removed instead?